.github/workflows/ci.yml

```diff
@@ -18,32 +18,25 @@ jobs:
       -
         name: Checkout
         uses: actions/checkout@v2
+      -
+        name: Install pipenv
+        run: pipx install pipenv
       -
         name: Set up Python
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      -
-        name: Get pip cache dir
-        id: pip-cache
-        run: |
-          echo "::set-output name=dir::$(pip cache dir)"
-      -
-        name: Persistent Github pip cache
-        uses: actions/cache@v2
-        with:
-          path: ${{ steps.pip-cache.outputs.dir }}
-          key: ${{ runner.os }}-pip3.8}
+          cache: "pipenv"
+          cache-dependency-path: 'Pipfile.lock'
       -
         name: Install dependencies
         run: |
-          pip install --upgrade pipenv
-          pipenv install --system --dev --ignore-pipfile
+          pipenv install --dev --ignore-pipfile
```

Comment:
Why not 
```suggestion
          pipenv sync --dev
```

Reply:
Right, I think that makes sense now the `--system` flag isn't being used. I suspect it was that way because `sync` doesn't include `--system`.

I'll change it to that shortly for these steps

Reply:
All finished.  It looks like `sync` does include the `--system` flag now, but we don't want that to allow the caching to function.